src/test/java/uk/gov/dhsc/htbhf/smartstub/helper/TestConstants.java

```diff
@@ -13,4 +13,7 @@
     public static final String ADDRESS_LINE_2 = "Mystery Spot";
     public static final String TOWN = "Springfield";
     public static final String POSTCODE = "AA1 1AA";
+
+    public static final String EMAIL = "test@email.com";
+    public static final String MOBILE = "07123456789";
```

Comment:
Again, I think `MOBILE_NUMBER` would be better.

Reply:
Changed to MOBILE_NUMBER